news/2024/2024-10-28-performance-points-star-rating-updates.md

```diff
@@ -147,11 +147,13 @@ Since the [last update to osu!taiko star rating 2 years ago](https://osu.ppy.sh/
 
 The previous system made to assess the stamina requirement for maps assumed two fingers per colour, leading to some impressive plays by players like [Ney](https://osu.ppy.sh/users/5991961) on [Mind Vortex - Alive](https://osu.ppy.sh/scores/1873110816) and others on [DJ Sharpnel - StrangeProgram](https://osu.ppy.sh/beatmapsets/5774#taiko/28065) becoming notorious for yielding high-pp scores.
 
-The [new system](https://github.com/ppy/osu/pull/20558) by [vun](https://osu.ppy.sh/users/6932501) introduces variable finger counts for stamina assessment. Mono-colour patterns that last more than 300 ms without a colour change are now considered to have four fingers available. This adjustment significantly nerfs certain maps and converts.
+The [new system](https://github.com/ppy/osu/pull/20558) by [vun](https://osu.ppy.sh/users/6932501) introduces variable finger counts for stamina assessment. colour patterns that last more than 300 ms without a colour change are now considered to have four fingers available. This adjustment significantly nerfs certain maps and converts. 
 
-Additionally, convert-specific nerfs have been removed from difficulty calculations to ensure that converts are weighted fairly against mode-specific maps. The changes to finger count availability also ensure that play styles using varying amounts of fingers per colour are fairly represented.
+To further combat TL-Tappong specific mapsets, a change by [Natelytle](https://osu.ppy.sh/users/17607667) introduces a new penalty factor that shows how much of a map's difficulty comes purely from single-hand (mono) patterns. For example, DJ Sharpnel - StrangeProgram has a factor of 99%, meaning 99% of its challenge is due to high-BPM, mono patterns.
```

Comment:
```suggestion
To further combat maps where TL-tapping is most effective, a change by [Natelytle](https://osu.ppy.sh/users/17607667) introduces a new penalty factor that shows how much of a map's difficulty comes purely from mono-colour patterns. For example, StrangeProgram has a factor of 99%, meaning 99% of its challenge is due to high-BPM mono patterns.
```
maybe? 
- a map shouldnt be defined by what strategy works best to play it (so not "TL-tapping specific" imo)
- strangeprogram artist not necessary since it's mentioned above and just reads poorly
- "mono-colour" is easy enough to understand even for not taiko player